image_search/config.yaml

```diff
@@ -0,0 +1,32 @@
+clip:
+  datasets:
+    datasets_path: ./data/datasets
+    dataset_name: images_dataset
+    dataset_description: Dataset for ImageSearch AISK
+    dataset_source_file: ./data/datasets/source.json
+    dataset_source_type: localMachine
+   
+  apps:
+    clip_app_id: 6c14325a-1be7-4e48-b38f-19b33745fc3b
+    application_field: vision
+   
+  urls:
+    base_url: https://sjc1-demo1.sambanova.net
```

Comment:
@jhpiedrahitao : Let's remove regerence to demo1

Reply:
Done 👍